scripts/configure-hypervisor.sh

```diff
@@ -362,8 +362,9 @@ generate_qemu_options() {
 	qemu_options+=(functionality:--enable-cap-ng)
 
 	# AVX2 is enabled by default, make sure it's on
```

Comment:
Comment is now not complete - could we embellish that with details about malloc-trim as well @devimc ?

Reply:
yes, see https://github.com/kata-containers/packaging/pull/463, thanks